module6/interviews/questions-4545.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Interview Questions
+---
+
+You can ask these questions in any order.  There are some questions each week that we know students will not be able to answer - this is by design to see how they exit a tough situation.  Keep track on notes on [this tracker](https://docs.google.com/spreadsheets/d/1eMtwW-06Wv0YAdAnZVm5k8_VFNKEKzQeq24kGxMTHbQ/edit?usp=sharing), please!
+
+
+## Week 1
+* Describe the 4 Pillars of OOP
+* What is HTTP and give a brief definition?
+* Explain Routing in .NET Core MVC?
+* What happens if the inherited interfaces have conflicting method names?
+* What is an enum in C#?
+* Difference between the Equality Operator (==) and Equals() Method in C#
+* Describe your background. How did you get to where you are now? Why coding?
+* Describe your strengths. How have you seen these in practice as a developer?
+* Describe a time when you received feedback that you were surprised by. What did you do with that feedback?
+* Tell me about a successful presentation you gave and why you think it was a hit.
+
```

Comment:
Because of the week's focus, I suggest we add "What questions do you have for me?"